File: ios/devices/DevicesInfo.swift
//
//  DevicesInfo.swift
//  wawajilive
//
//  Created by 张继伟 on 2017/12/21.
//  Copyright © 2017年 Facebook. All rights reserved.
//

import Foundation

class DevicesInfo : Codable{
  var devicesId: String
  var versionCode: String
  var versionName: String
  var channel: String
  init(devicesId: String,versionCode: String,versionName: String,channel: String) {
    self.devicesId = devicesId
    self.versionCode = versionCode
    self.versionName = versionName
    self.channel = channel
  }
}

File: android/app/src/main/java/com/starcandy/wawaji/devices/DevicesInfo.java
package com.starcandy.wawaji.devices;

/**
 * Created by zhangjiwei on 2017/12/21.
 */

public class DevicesInfo {
	private String devicesId;
	private String versionCode;
	private String versionName;
	private String channel;
	
	
	public String getDevicesId() {
		return devicesId;
	}
	
	public void setDevicesId(String devicesId) {
		this.devicesId = devicesId;
	}
	
	public String getVersionCode() {
		return versionCode;
	}
	
	public void setVersionCode(String versionCode) {
		this.versionCode = versionCode;
	}
	
	public String getVersionName() {
		return versionName;
	}
	
	public void setVersionName(String versionName) {
		this.versionName = versionName;
	}
	
	public String getChannel() {
		return channel;
	}
	
	public void setChannel(String channel) {
		this.channel = channel;
	}
	
	@Override
	public String toString() {
		return "DevicesInfo{" +
				"devicesId='" + devicesId + '\'' +
				", versionCode='" + versionCode + '\'' +
				", versionName='" + versionName + '\'' +
				", channel='" + channel + '\'' +
				'}';
	}
}
